src/components/Resource/ResourceDetails.tsx

```diff
@@ -27,25 +22,6 @@ export default function ResourceDetails(props: { id: string }) {
       }
     },
   });
-
-  const handleResourceDelete = async () => {
-    setOpenDeleteResourceDialog(true);
-    const { res, data } = await request(routes.deleteResourceRecord, {
```

Comment:
do delete the route definitions too since such api's doesn't exist.

Reply:
Sure, I'll